### PR TITLE
Add instructions for `apt-get` install to `README`

### PR DESCRIPTION
**Pull request by ldennington**
Updating README.md with instructions for apt-get setup and install
for Ubuntu Bionic and Hirsute.

### CODE REVIEW

**Review by mjcheetham — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -88,6 +88,30 @@ sudo /usr/local/share/gcm-core/uninstall.sh
...
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+#### Bionic
+
+```shell
+curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod
+sudo apt-get update
+sudo apt-get install microsoft-git
```
Comment (derrickstolee) [inline README.md:101]: This package name should be different for GCM core.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,30 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ### Linux Debian package (.deb)
 
+`apt-get` support is available for Ubuntu Bionic Beaver (18.04) and Hirsute 
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+#### Bionic
```
Comment (derrickstolee) [inline README.md:95]: I mentioned in your `microsoft/git` PR that these titles might be easier to read in a scan if they were `Ubuntu 18.04 (Bionic)` and `Ubuntu 21.04 (Hirsute)`.

**Review by derrickstolee — CHANGES_REQUESTED** [review]
Blocking on the `apt-get install` command.

File: README.md (hunk 1 above)
Comment (ldennington) [reply README.md:101]: Oh dear.

**Review by derrickstolee — APPROVED** [review]

**Pull request merged**